Updates for 2023 (#5)
RoboJackets/membership-agreement: agreement.md

### Dress for the work being performed

- You must wear personal protection equipment (safety glasses, earplugs, breathing masks, etc.) anywhere signs are
posted in the SCC, or whenever appropriate based on good training for that tool. For example, you must wear safety
glasses whenever powered, sharp, or impact tools are being used in your vicinity.
- Never approach rotating or large power tools with loose clothing, long untied hair, lanyards, headphone cords around
your neck, or large jewelry that could pull you toward the machine.
- Shoes worn in the fabrication spaces must cover the tops of your feet.

### The buddy system applies whenever you’re using powered tools anywhere in the SCC

- Work with at least one person within voice and visual distance of you when working in fabrication spaces.

> - You must not be separated by a closed door from your buddy.

Reviewer: ```suggestion
- You must not be separated from your buddy by a closed door.
```

- Exits and walkways must be kept clear of furniture or workpieces for safe passage during an emergency.
- Never modify any tool, and especially, never remove safety devices from tools.
- If you see someone working in an unsafe manner, you must gently correct them as a colleague. If you feel uncomfortable

> doing so, contact your team leadership, Jacob Tompkins, or Dr. Mayor, and inform them of the situation.

Reviewer 2: Would it be any better to refer to these individuals by their position (whatever titles Jake and Dr. Mayor hold w/r/t RJ/SCC) rather than by name or at least include them? This may help newer members understand _why_ these people are specifically called out here rather than random names with no association. For example, `Jacob Tomkins (SCC Manager), Dr. Mayor (???)`.

Author: Added titles

immediate area capable of and trained to safely shut down any power equipment you might be using. At least one other
person must be within talking distance when any powered, sharp, or impact tools are being used. No logged-in machine
may be left unattended at any time.
3. Active use of the Common Machining Area, and active noise-generating fabrication activities in team spaces, should not
occur during announced "cold shop" events, such as sponsor visits to the SCC. Such activities may resume after the end
of the scheduled visit, even if the sponsor is still present. You must apply all safety precautions with sponsors if
they enter active work areas.
4. Safety glasses must be worn by everyone in the work area anytime that powered, sharp, or impact tools are being used.
Respirators must be worn while cutting, sanding, or painting with materials prone to cause lung damage. Ear protection
must be worn during times when the noise level is especially high or prolonged.
5. Sandals or other open toe shoes are not allowed on the work floor of the shop at any time. You may wear sandals while
working in the offices and on a reasonable path from the door to the offices only.

> 6. Flammables must be in sealed, correctly labeled containers, and inside a marked flammables cabinet. Containers capable

Reviewer: ```suggestion
6. Flammables must be in sealed, correctly labeled containers inside a marked flammables cabinet. Containers capable
```

13. The horizontal surface over the entrance hallway must not be used for storage. The un-fenced areas on the mezzanine
must remain clear at all times.
14. Personal or classroom projects must not be a distraction or take up space in any common area of the shop, or in the
parking or drive areas outside the shop, without explicit permission from Dr. Mayor. Personal projects must not make
any significant use of team or SCC consumable resources.
15. Any time a vehicle is parked in front of the roll-up door, the keys must be inside the building with someone who has
permission to move the vehicle at a moment's notice. Contact information for that individual must be prominently
visible through the windshield.

Infractions of these rules may lead to individual and/or team sanctions, up to and including expulsion from the SCC.

If you need clarification on any of these rules, contact [scc@me.gatech.edu][sccrt].

## Student Competition Center Building Access Policy

All teams and visitors within the Student Competition Center ("SCC") must follow these guidelines. Failure to follow
these guidelines may result in removal of access to the space.

> 1. Members with BuzzCard access to the building may not let other members into the space, via car gate or door access,

Reviewer 2: ```suggestion
1. Members with BuzzCard access to the building may not let other members into the space, via vehicle gate or door access,
```

> without getting explicit permission from the security desk, building management, Jacob Tompkins, or Dr. Rhett Mayor.

Reviewer 2: ```suggestion
without getting explicit permission from the security desk, building management, Jacob Tompkins, or Dr. Mayor.
```
Consistent use of Dr. Mayor without a first name

> 2. Every vehicle must buzz in via BuzzCard. Multiple vehicles may not enter the parking lot during the same tap, even
> if the second vehicle has BuzzCard access.

Reviewer 2: ```suggestion
2. Every vehicle must be granted access to the parking lot by presenting a valid BuzzCard to the card reader. 
Tailgating behind another vehicle to gain entry is prohibited, even if the second vehicle would otherwise have access.
```

3. People without BuzzCard access must sign in at the security desk. This includes new members who have not yet
received BuzzCard access, visitors, professors, or sponsors.
4. Members of the SCC and visitors must only use the approved spaces listed below. If members are found wandering the
space or attempting to access labs or other rooms, their access may be suspended.
a. Main SCC area
b. RoboJackets space
c. Kitchenette
d. Conference rooms 1271, 1253, 1257, 1102, 1103, and the Boeing classroom

> Direct any questions to the SCCGB officers, Jacob Tompkins, or Dr. Rhett Mayor.

Reviewer 2: ```suggestion
Direct any questions to the SCCGB officers, Jacob Tompkins, or Dr. Mayor.
```
Consistent use of Dr. Mayor without a first name
